xrpl/utils/txn_parser/get_final_balances.py

```diff
@@ -0,0 +1,59 @@
+"""Parse final balances of every account involved in the given transaction."""
+
+from decimal import Decimal
+from typing import List, Optional
+
+from xrpl.models import TransactionMetadata
+from xrpl.utils.txn_parser.utils import (
+    ComputedBalances,
+    NormalizedNode,
+    get_node_balance,
+    get_value,
+    group_by_account,
+    normalize_nodes,
+)
+
+
+def get_final_balances(metadata: TransactionMetadata) -> List[ComputedBalances]:
```

Comment:
This seems like basically the same function as before - would it be helpful to abstract this into a helper function that accepted the "parse function" as a parameter? (Ex. `_compute_final_balance` in this case)

Reply:
So you want a function like `get_balances` that takes, let's say an Enum like `BalanceParser.BalanceChanges` and `BalanceParser.FinalBalances` that would link to the `_compute_balance_change` and `_compute_final_balance` functions? That would not be backwards compatible, right?

Reply:
Makes sense 👍 But I would name it derive_account_balances, because the function itself does not group anything.

Reply:
That's a better name for sure :)